lib/galaxy/jobs/runners/kubernetes.py

```diff
@@ -543,23 +535,27 @@ def fail_job(self, job_state):
 
     def __produce_log_file(self, job_state):
         pod_r = Pod.objects(self._pykube_api).filter(selector="app=" + job_state.job_id)
-        logs = ""
+        log_string = ""
         for pod_obj in pod_r.response['items']:
             try:
                 pod = Pod(self._pykube_api, pod_obj)
-                logs += "\n\n==== Pod " + pod.name + " log start ====\n\n"
-                logs += pod.logs(timestamps=True)
-                logs += "\n\n==== Pod " + pod.name + " log end   ===="
+                log_string += "\n\n==== Pod " + pod.name + " log start ====\n\n"
+                log_string += pod.logs(timestamps=True)
+                log_string += "\n\n==== Pod " + pod.name + " log end   ===="
             except Exception as detail:
-                log.info("Could not write pod\'s " + pod_obj['metadata']['name'] +
-                         " log file due to HTTPError " + str(detail))
+                log.info("Could not write log file for pod %s due to HTTPError %s",
+                         detail, pod_obj['metadata']['name'])
```

Comment:
I think that you have objects order swapped here, pod_obj['metadata']['name'] should go first if I'm not mistaken.

Reply:
Thank you  @pcm32 :bowing_man: